java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -824,6 +824,52 @@ public void sadd_srem_scard_smembers_key_with_non_set_value(BaseClient client) {
         assertTrue(e.getCause() instanceof RequestException);
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void smove(BaseClient client) {
+        String setKey1 = "{key}" + UUID.randomUUID();
+        String setKey2 = "{key}" + UUID.randomUUID();
+        String setKey3 = "{key}" + UUID.randomUUID();
+        String nonSetKey = "{key}" + UUID.randomUUID();
+
+        assertEquals(3, client.sadd(setKey1, new String[] {"1", "2", "3"}).get());
+        assertEquals(2, client.sadd(setKey2, new String[] {"2", "3"}).get());
+        // move an elem
+        assertTrue(client.smove(setKey1, setKey2, "1").get());
+        assertEquals(Set.of("2", "3"), client.smembers(setKey1).get());
+        assertEquals(Set.of("1", "2", "3"), client.smembers(setKey2).get());
+        // move an elem which preset at destination
```

Comment:
I doubt that you need this test case.  // move an elem which is preset at the destination (you are testing how sets actually work here)

Reply:
True, but it does not hurt.
It also shows use cases of the command. I like when IT could be interpreted as examples.